src/utils/url.js

```diff
@@ -157,3 +157,12 @@ export const autocompleteUrl = (
         value.indexOf('.') === -1 ? append : !value.match(/.+\..+\.+./g) ? shortAppend : ''
       }`
     : '';
+
+export const encodeImageUri = (uri: string): string => {
+  const last = uri.lastIndexOf('/');
+  const imageName = uri.substring(last + 1);
+  const original = uri.substring(0, last);
```

Comment:
Is `original` a good name to represent that part of the string?
Is it maybe `path`?

Reply:
Yeah, `path` sounds good. :)